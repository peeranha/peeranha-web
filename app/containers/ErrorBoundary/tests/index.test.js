import React from 'react';
import { ErrorBoundary } from '../index';

const children = <div>Children</div>;
React.Children.only = jest.fn().mockImplementation(() => children);

const cmp = new ErrorBoundary();

cmp.state = {
  error: {},
  errorInfo: {},
};

cmp.props = {
  locale: 'en',
  children: 'children',
};

describe('<ErrorBoundary />', () => {
  describe('snapshot test', () => {
    it('error is null', () => {
      cmp.state.error = null;
      cmp.state.errorInfo = null;
      expect(cmp.render()).toMatchSnapshot();
    });

    it('error is not null', () => {
      cmp.state.error = {};
      cmp.state.errorInfo = {};
      expect(cmp.render()).toMatchSnapshot();
    });
  });

  describe('reloadApp', () => {
    it('test', () => {
      cmp.reloadApp();
      expect(window.location.pathname).toBe('/');
    });
  });

  describe('componentDidCatch', () => {
    const error = {};
    const errorInfo = {};
    cmp.setState = jest.fn();

    expect(cmp.setState).toHaveBeenCalledTimes(0);

    cmp.componentDidCatch(error, errorInfo);
    expect(cmp.setState).toHaveBeenCalledTimes(1);
    expect(cmp.setState).toHaveBeenCalledWith({ error, errorInfo });
  });
});
