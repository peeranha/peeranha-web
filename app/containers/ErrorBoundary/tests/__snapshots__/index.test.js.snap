// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorBoundary /> snapshot test error is not null 1`] = `
<React.Fragment>
  <div>
    <HelmetWrapper
      defer={true}
      encodeSpecialCharacters={true}
    >
      <title>
        Error
      </title>
      <meta
        content="Description of error boundary"
        name="description"
      />
    </HelmetWrapper>
    <UNDEFINED
      error={Object {}}
      errorInfo={Object {}}
      reloadApp={[Function]}
      translations={
        Object {
          "app.common.about": "About",
          "app.common.add": "Add",
          "app.common.addQuestion": "Add question",
          "app.common.ago": "ago",
          "app.common.all": "All",
          "app.common.allCommunities": "All communities",
          "app.common.allQuestions": "All questions",
          "app.common.answered": "Answered",
          "app.common.answers": "Answers",
          "app.common.asked": "Asked",
          "app.common.cancel": "Cancel",
          "app.common.close": "Close",
          "app.common.communities": "Communities",
          "app.common.contacts": "Contacts",
          "app.common.edit": "Edit",
          "app.common.faq": "FAQ",
          "app.common.login": "Log in",
          "app.common.logout": "Log out",
          "app.common.manageMyComm": "Manage my communities",
          "app.common.myFeed": "My feed",
          "app.common.nothingToSeeYet": "Nothing to see yet",
          "app.common.peerania": "Peerania",
          "app.common.posts": "Posts",
          "app.common.privacyPolicy": "Privacy Policy",
          "app.common.profile": "Profile",
          "app.common.questions": "Questions",
          "app.common.search": "Search",
          "app.common.see": "See",
          "app.common.settings": "Settings",
          "app.common.signUp": "Sign Up",
          "app.common.subscribeMore": "Subscribe more",
          "app.common.support": "Support",
          "app.common.tags": "Tags",
          "app.common.top": "Top",
          "app.common.upload": "Upload",
          "app.common.users": "Users",
          "app.common.you": "You",
          "app.common.youAsked": "You asked",
          "app.components.Footer.header": "Copyright: Peerania",
          "app.components.FormFields.fileSize": "File size is exceeded",
          "app.components.FormFields.requiredField": "Required field",
          "app.components.FormFields.wrongEmail": "Wrong email",
          "app.components.FormFields.wrongLength15x100": "Value must be at least 15 characters and no longer than 100 characters",
          "app.components.FormFields.wrongLength1x5": "Value must be at least 1 characters and no longer than 5 characters",
          "app.components.FormFields.wrongLength20x1000": "Value must be at least 20 characters and no longer than 1000 characters",
          "app.components.FormFields.wrongLength25x30000": "Value must be at least 25 characters and no longer than 30000 characters",
          "app.components.FormFields.wrongLength2x15": "Value must be at least 2 characters and no longer than 15 characters",
          "app.components.FormFields.wrongLength30000": "Maximum number of characters exceeded",
          "app.components.FormFields.wrongLength3x20": "Value must be at least 3 characters and no longer than 20 characters",
          "app.components.NoAccess.errorMessage": "No access to complete this action!",
          "app.components.RatingStatus.banned": "Banned",
          "app.components.RatingStatus.heroResident": "Hero Resident",
          "app.components.RatingStatus.jrResident": "Junior Resident",
          "app.components.RatingStatus.legResident": "Legendary Resident",
          "app.components.RatingStatus.newbie": "Newbie",
          "app.components.RatingStatus.resident": "Resident",
          "app.components.RatingStatus.srResident": "Senior Resident",
          "app.components.RatingStatus.stranger": "Stranger",
          "app.containers.AskQuestion.description": "Ask question | Description",
          "app.containers.AskQuestion.notEnoughRating": "To complete this action, your rating has to be more than",
          "app.containers.AskQuestion.postQuestion": "Post question",
          "app.containers.AskQuestion.title": "Ask question",
          "app.containers.Communities.description": "Communities | Description",
          "app.containers.Communities.tags": "Tags",
          "app.containers.Communities.title": "Communities",
          "app.containers.CreateCommunity.avatar": "Avatar",
          "app.containers.CreateCommunity.createCommunity": "Create community",
          "app.containers.CreateCommunity.description": "Description of CreateCommunity",
          "app.containers.CreateCommunity.descriptionField": "Description",
          "app.containers.CreateCommunity.name": "Name",
          "app.containers.CreateCommunity.title": "Community | Create",
          "app.containers.CreateTag.createTag": "Create tag",
          "app.containers.CreateTag.description": "Create tag | Description",
          "app.containers.CreateTag.descriptionField": "Description",
          "app.containers.CreateTag.name": "Name",
          "app.containers.CreateTag.title": "Create tag",
          "app.containers.EditAnswer.description": "Edit Answer Description",
          "app.containers.EditAnswer.submitButtonName": "Save",
          "app.containers.EditAnswer.title": "Edit Answer",
          "app.containers.EditQuestion.description": "Edit Question Description",
          "app.containers.EditQuestion.submitButtonName": "Save",
          "app.containers.EditQuestion.title": "Edit question",
          "app.containers.ErrorBoundary.description": "Description of error boundary",
          "app.containers.ErrorBoundary.details": "Details",
          "app.containers.ErrorBoundary.problemWithWebpage": "A problem occurred with this webpage",
          "app.containers.ErrorBoundary.reloadPage": "Reload page",
          "app.containers.ErrorBoundary.title": "Error",
          "app.containers.HomePage.about": "About",
          "app.containers.HomePage.askQuestion": "Ask question",
          "app.containers.HomePage.askQuestionAndGetHelp": "Peerania has passionate experts from all areas of life ready to help others. Users can return the favor by helping thousands of others answer questions they may also have. Users can get the answers to their questions in a matter of minutes.",
          "app.containers.HomePage.beTheFirst": "Joining early means enjoying higher levels of rewards",
          "app.containers.HomePage.description": "Decentralized questions and answers website that rewards users with crypto tokens for their contributions.",
          "app.containers.HomePage.differentFromOtherSites": "How is Peerania different from other questions and answers websites?",
          "app.containers.HomePage.differentFromOtherSitesCollapsed": "Unlike other question and answer websites where an organization owns the website and all the data generated from users, Peerania is owned by the community and rewards are distributed to community members for their contributions and the attention they give to the website.",
          "app.containers.HomePage.differentThan": "Peerania is more than a Q&A website.",
          "app.containers.HomePage.differentThan2": "It is a community that puts users at the heart by:",
          "app.containers.HomePage.earlyAdopters": "Valuing Early Adopters",
          "app.containers.HomePage.earnReputation": "Earning Reputation",
          "app.containers.HomePage.email": "Email",
          "app.containers.HomePage.faq": "FAQ",
          "app.containers.HomePage.faqDescription": "Peerania | FAQ Description",
          "app.containers.HomePage.faqTitle": "Peerania | FAQ",
          "app.containers.HomePage.getHelp": "Getting Help from the Community",
          "app.containers.HomePage.getMoreAnswers": "Get more answers",
          "app.containers.HomePage.getPaidInCrypto": "Receiving Crypto Token Rewards",
          "app.containers.HomePage.getPaidInCryptoFromWeekly": "A weekly pool of reward tokens is distributed to users for their contributions. Contributions include asking and answering questions. These reward tokens can be sold on the open market to anyone wanting to pay for content to be placed on the platform. This paid content may include product advertisements or job openings. Users will always be rewarded for their valuable contributions. ",
          "app.containers.HomePage.getReward": "Sign Up",
          "app.containers.HomePage.getStarted": "Get started",
          "app.containers.HomePage.helpCommunity": "Experts contribute to the community by answering questions from their peers. Answers not only help the person who asked the question but thousands of others who have similar questions. Once a question has been answered, the knowledge can be shared with current and future users.",
          "app.containers.HomePage.helpfulAndGenerous": "Helpful and generous",
          "app.containers.HomePage.honorableAndFascinating": "Honorable and fascinating",
          "app.containers.HomePage.howEarn": "How can I earn crypto tokens on Peerania?",
          "app.containers.HomePage.howEarnCollapsed": "All activities on the website that are considered valuable for the community are rewarded in crypto tokens.  Contributions that earn tokens may include posting a detailed question, providing a helpful answer and participating in moderation.",
          "app.containers.HomePage.howItWorks": "How does it work?",
          "app.containers.HomePage.howItWorksCollapsed": "Peerania is built using smart contracts on the EOS blockchain. Large content is stored via distributed storage (IPFS) and only hashes of all documents are posted on the blockchain to save expensive blockchain storage. Reputation, privileges, moderation, rewards and all other rules are built into smart contracts and are visible to anyone.",
          "app.containers.HomePage.learnMore": "Where can I learn more?",
          "app.containers.HomePage.learnMoreCollapsed": "The best ways to learn more about to join our twitter, read our blog or send us an email to support@peerania.com .",
          "app.containers.HomePage.login": "Log in",
          "app.containers.HomePage.message": "Message",
          "app.containers.HomePage.messageHasNotBeenSent": "Something went wrong and your message has not been sent. Please try again.",
          "app.containers.HomePage.nikitaSyr": "Nikita Syrovatnikov",
          "app.containers.HomePage.nikitaSyrDescription": "Blockchain app technician, interesting in solving different problems and learning new technologies. I believe that any problem can be solved with the right approach.",
          "app.containers.HomePage.nikitaSyrRole": "Back-end and Smart-contract Developer",
          "app.containers.HomePage.none": "None",
          "app.containers.HomePage.ourTeam": "Our team",
          "app.containers.HomePage.ownershipByUsers": "Ownership by Users",
          "app.containers.HomePage.participateInModeration": "Governing the community is in the hands of the members. Making the best decisions for the platform result in a higher token value. Through the governance structure built into Peerania, the website is moderated by its users. As a decentralized, open source project, the future is decided by you the user. Much like a local community, your voice matters.",
          "app.containers.HomePage.peeraniaDecentralized": "Peerania is truly permissionless and decentralized. Website governance is decided for the community, by the community. Peerania puts power in the hands of the contributors, not the developers.",
          "app.containers.HomePage.peeraniaPhilosophy": "Peerania’s philosophy is to reward early users generously for their time. As the website grows, rewards will even out and be distributed to more members of the community.",
          "app.containers.HomePage.peopleSaveHours": "Millions of people save hours of work through the knowledge of others on Q&A websites on a daily basis. Peerania’s users are rewarded with PEER crypto tokens for valuable activities.",
          "app.containers.HomePage.platformUnderDeveloping": "The website is currently under development. Sign up for notifications to be one of the first to join and earn higher rewards as an early adopter.",
          "app.containers.HomePage.profitableAndModern": "Profitable and modern",
          "app.containers.HomePage.review": "Review",
          "app.containers.HomePage.rewardingUser": "Rewarding Users with Crypto",
          "app.containers.HomePage.rewards": "Rewards",
          "app.containers.HomePage.rewardsPool": "Rewards are distributed weekly to all contributors.  As the number of weekly contributors goes up, rewards per user go down over time. Joining early means enjoying higher levels of rewards for your pioneering contributions to an early community.",
          "app.containers.HomePage.romRem": "Roman Remniov",
          "app.containers.HomePage.romRemDescription": "Based in Minsk front-end developer, currently focused on data-management and User Experience in decentralized applications.",
          "app.containers.HomePage.romRemRole": "Front-end Developer",
          "app.containers.HomePage.ruleThePlatform": "Governing the Community",
          "app.containers.HomePage.sendMessage": "Send message",
          "app.containers.HomePage.sergeyIlin": "Sergey Ilin",
          "app.containers.HomePage.sergeyIlinDescription": "Developer and architect with over decade of experience building software from websites to large scale enterprise solutions for Fortune 500 companies. Passionate about finding groundbreaking practical applications of Blockchain technology.",
          "app.containers.HomePage.sergeyIlinRole": "Co-founder, CTO",
          "app.containers.HomePage.shareKnowledge": "Sharing Knowledge",
          "app.containers.HomePage.signUpFree": "Sign Up",
          "app.containers.HomePage.simpleAndAffordable": "Simple and affordable",
          "app.containers.HomePage.steveKreynin": "Steve Kreynin",
          "app.containers.HomePage.steveKreyninDescription": "Successful serial entrepreneur. CEO of Forte Group, an international IT Consulting Company.",
          "app.containers.HomePage.steveKreyninRole": "Co-founder, President",
          "app.containers.HomePage.subject": "Subject",
          "app.containers.HomePage.systemError": "System error",
          "app.containers.HomePage.team": "Team",
          "app.containers.HomePage.title": "Peerania - Decentralized Q&A website with crypto rewards",
          "app.containers.HomePage.tokensReward": "Where do tokens for rewards come from?",
          "app.containers.HomePage.tokensRewardCollapsed": "Peerania continually creates new tokens once a week until the total supply of tokens is reached. For a currency, this is known as inflation. Every week, newly created tokens are transferred to the reward pool. This weekly reward pool is distributed proportionally to users’ contributions to the community during the week. As the website and community grow, additional sources begin to contribute tokens to the weekly reward pool. Such sources can be fees for the ads and job postings published on the website, donations, or other ways that the community decides to monetize the website. The rate of new token creation reduces every year and eventually decreases to zero. At this point, the weekly reward pool will solely consist of tokens monetized by the website.",
          "app.containers.HomePage.ulyanaPopova": "Ulyana Popova",
          "app.containers.HomePage.ulyanaPopovaDescription": "Ready to produce wire-frames, mock-ups, and prototypes which implement the modernization of systems with an effective, efficient, aesthetic, and uncluttered user experience. Able to use humour and energy to keep the momentum going.",
          "app.containers.HomePage.ulyanaPopovaRole": "UX and UI Designer",
          "app.containers.HomePage.valueOfToken": "Where does the value of the token come from?",
          "app.containers.HomePage.valueOfTokenCollapsed": "Because Peerania is blockchain based, tokens can be traded in the open market. Additionally, Peerania incorporates game elements and users unlock additional powers by using the tokens. Finally, the token will be used as a payment method on the website for bounties to answer questions. Fees in the form of tokens will be required to publish job postings and ads on the website or any other paid services that website will provide.",
          "app.containers.HomePage.weAppreciate": "We highly appreciate your feedback, opinion or suggestion for the platform. Send us few lines or even better let us know if you have 15 minutes to chat.",
          "app.containers.HomePage.weAreDecentralized": "Decentralized questions and answers website that rewards users with crypto tokens for their contributions.",
          "app.containers.HomePage.websiteLive": "Is the website live?",
          "app.containers.HomePage.websiteLiveCollapsed": "The website is in active development since June of 2018. A beta release is planned for the end of Q2 2019.",
          "app.containers.HomePage.whatBlockchain": "What Blockchain Peerania is Built on?",
          "app.containers.HomePage.whatBlockchainCollapsed": "Peerania is built on the EOS blockchain. EOS provides short confirmation times for transactions and does not require users to pay any transaction fees.",
          "app.containers.HomePage.whatIsCryptocurrency": "What is the cryptocurrency of Peerania?",
          "app.containers.HomePage.whatIsCryptocurrencyCollapsed": "Peerania uses its own crypto-token (or cryptocurrency) called PEER.",
          "app.containers.HomePage.whatIsPeerania": "What is Peerania?",
          "app.containers.HomePage.whatIsPeeraniaCollapsed": "Peerania is a decentralized question and answer website that rewards users with crypto tokens for their valuable contributions. Peerania is built on a blockchain and it is owned by the community rather than by a single organization.",
          "app.containers.HomePage.whetherYouAsk": "As users build their level of contributions, they also improve their reputation. The community recognizes valuable contributors who answer questions for others. Earning a high reputation unlocks privileges as a user’s standing within the community increases!",
          "app.containers.HomePage.whitePaper": "Do you have white paper?",
          "app.containers.HomePage.whitePaperCollapsed": "No. We are an agile team. We have a high-level vision but detailed specifications change frequently during the development process. We will create detailed documentation when a beta version of the product it ready for the public.",
          "app.containers.HomePage.worthyAndPrestig": "Worthy and prestigious",
          "app.containers.HomePage.yourContributionsRewarded": "Your Contributions Rewarded",
          "app.containers.HomePage.yourEmail": "Your email address",
          "app.containers.HomePage.yourEmailIsRegistred": "The website is under development but your email is registered. You will receive an email with the steps to continue registration when the website is ready.",
          "app.containers.HomePage.yourName": "Your name",
          "app.containers.Login.doNotHaveAcc": "Do not have an account?",
          "app.containers.Login.header": "No Peerania account exists for selected identity.",
          "app.containers.Login.loginWithScatter": "Log in with Scatter",
          "app.containers.Login.requirementToSignUp": "Sign Up for Peerania account",
          "app.containers.Login.selectAnotherIdentity": "Select another identity",
          "app.containers.Login.signUp": "Sign up",
          "app.containers.Other.errorMessage": "An error occurred",
          "app.containers.Other.successMessage": "Action has been completed successfully",
          "app.containers.Profile.aboutLabel": "About me",
          "app.containers.Profile.activity": "Activity",
          "app.containers.Profile.answered": "answered",
          "app.containers.Profile.companyLabel": "Company",
          "app.containers.Profile.companyTip": "Company tip...",
          "app.containers.Profile.displayNameLabel": "Display name",
          "app.containers.Profile.displayNameTip": "Display name tip...",
          "app.containers.Profile.editProfile": "Edit profile",
          "app.containers.Profile.heHasnoAnswersAndQuestions": "He has no answers and questions.",
          "app.containers.Profile.informationIsBlank": "Information about you is currently blank.",
          "app.containers.Profile.lastAnswer": "Last answer",
          "app.containers.Profile.locationLabel": "Location",
          "app.containers.Profile.locationTip": "Location tip...",
          "app.containers.Profile.memberSince": "Member since ",
          "app.containers.Profile.noAnswersYet": "No answers yet",
          "app.containers.Profile.positionLabel": "Position",
          "app.containers.Profile.positionTip": "Position tip...",
          "app.containers.Profile.profile": "Profile",
          "app.containers.Profile.profileDescription": "Profile description",
          "app.containers.Profile.reputation": "Reputation",
          "app.containers.Profile.risen": "Risen",
          "app.containers.Profile.saveButton": "Save",
          "app.containers.Profile.userHasntActivityYet": "This user hasn’t activity yet",
          "app.containers.Profile.wrongUser": "Wrong user",
          "app.containers.QuestionForm.addForLineBreaks": "for linebreak add 2 spaces at end",
          "app.containers.QuestionForm.backtipEscapes": "backtick escapes \`like _so_\`",
          "app.containers.QuestionForm.communityLabel": "Community",
          "app.containers.QuestionForm.communityTip": "Community tip...",
          "app.containers.QuestionForm.contentLabel": "Body",
          "app.containers.QuestionForm.howToAsk": "How to ask?",
          "app.containers.QuestionForm.howToFormat": "How to format?",
          "app.containers.QuestionForm.howToTag": "How to tag?",
          "app.containers.QuestionForm.indentCode": "indent code by 4 spaces",
          "app.containers.QuestionForm.italicAndBold": "_italic_ or **bold**",
          "app.containers.QuestionForm.previewLabel": "Preview",
          "app.containers.QuestionForm.putReturnsBetweenParagraphs": "put returns between paragraphs",
          "app.containers.QuestionForm.quoteByPlacing": "quote by placing &gt; at start of line",
          "app.containers.QuestionForm.tagsLabel": "Tags",
          "app.containers.QuestionForm.tagsTip": "Tags tip...",
          "app.containers.QuestionForm.tips": "Tips",
          "app.containers.QuestionForm.title": "Ask question",
          "app.containers.QuestionForm.titleLabel": "Title",
          "app.containers.QuestionForm.titleTip": "Title tip...",
          "app.containers.QuestionForm.toMakeLinks": "to make links &lt;http://foo.com&gt;[foo](http://foo.com)&lt;a href=&quot;http://foo.com&quot;&gt;fo",
          "app.containers.Questions.allCommunities": "All communities",
          "app.containers.Questions.noQuestions": "No questions",
          "app.containers.Questions.noQuestionsYet": "No questions yet.",
          "app.containers.Questions.readMyFeed": "Read my feed",
          "app.containers.Questions.subscribeToCommToKeep": "Subscribe to communities to keep track of topics that you are interested in.",
          "app.containers.Questions.thisIsNewCommunity": "This is a new community, you can be the first and your question may be the most popular.",
          "app.containers.Questions.title": "Questions",
          "app.containers.Questions.youDontHaveFeedToRead": "You don't have any feed to read",
          "app.containers.ScatterInstaller.backToOptions": "Back to options",
          "app.containers.ScatterInstaller.header": "To complete this action in our app you need to install",
          "app.containers.ScatterInstaller.nextstep": "Next step",
          "app.containers.SelectAccount.backToOptions": "Back to options",
          "app.containers.SelectAccount.header": "Scatter account was not selected.",
          "app.containers.SelectAccount.selectIdent": "Select identity",
          "app.containers.SignUp.alreadyHaveAcc": "Already have Peerania account?",
          "app.containers.SignUp.backToSignUpOptions": "Back to Sign Up options",
          "app.containers.SignUp.displayName": "Display Name",
          "app.containers.SignUp.eosAccount": "EOS Account",
          "app.containers.SignUp.identity": "identity",
          "app.containers.SignUp.isAlreadyRegistred": "is already registred",
          "app.containers.SignUp.login": "Log in",
          "app.containers.SignUp.loginWith": "Login with",
          "app.containers.SignUp.or": "or",
          "app.containers.SignUp.selectedIdentity": "Selected identity",
          "app.containers.SignUp.serverMessage": "Error",
          "app.containers.SignUp.signUp": "Sign Up",
          "app.containers.SignUp.signUpDescription": "Description of Sign Up",
          "app.containers.SignUp.signUpWith": "Sign up with Scatter",
          "app.containers.SuggestedCommunities.creatorCannotUpvoteOwnComm": "Creator cannot upvote own community",
          "app.containers.SuggestedCommunities.description": "Suggested communities | Description",
          "app.containers.SuggestedCommunities.noSuggestedCommunities": "There are no suggested communities",
          "app.containers.SuggestedCommunities.title": "Suggested communities",
          "app.containers.SuggestedCommunities.youHaveAlreadyVoted": "You have already voted",
          "app.containers.SuggestedTags.creatorCannotUpvoteOwnComm": "Creator cannot upvote own tag",
          "app.containers.SuggestedTags.description": "Suggested tags | Description",
          "app.containers.SuggestedTags.noSuggestedTags": "There are no suggested tags",
          "app.containers.SuggestedTags.title": "Suggested tags",
          "app.containers.SuggestedTags.youHaveAlreadyVoted": "You have already voted",
          "app.containers.Tags.createTag": "Create tag",
          "app.containers.Tags.description": "Description of Tags",
          "app.containers.Tags.suggestedTags": "Suggested tags",
          "app.containers.Tags.title": "Tags",
          "app.containers.UserProfileNav.profile": "Profile",
          "app.containers.ViewQuestion.addComment": "Add comment",
          "app.containers.ViewQuestion.alreadyAnswered": "You already answered this question",
          "app.containers.ViewQuestion.answerIsCorrect": "You cannot delete it because your answer is marked as correct",
          "app.containers.ViewQuestion.answers": "answers",
          "app.containers.ViewQuestion.commentButton": "Comment",
          "app.containers.ViewQuestion.communityChoice": "Community choice",
          "app.containers.ViewQuestion.deleteButton": "Delete",
          "app.containers.ViewQuestion.description": "Question's description",
          "app.containers.ViewQuestion.editButton": "Edit",
          "app.containers.ViewQuestion.itemsMax": "Number of items is achieved of maximum",
          "app.containers.ViewQuestion.lastEdited": "Last edited",
          "app.containers.ViewQuestion.moreComments": "More comments",
          "app.containers.ViewQuestion.noRootsToVote": "You cannot vote to own questions and answers",
          "app.containers.ViewQuestion.notEnoughModPoints": "To complete this action, your moderation points rating has to be more than",
          "app.containers.ViewQuestion.notEnoughRating": "To complete this action, your rating has to be more than",
          "app.containers.ViewQuestion.postAnswerButton": "Post answer",
          "app.containers.ViewQuestion.postCommentButton": "Post comment",
          "app.containers.ViewQuestion.questionNotExist": "This question does not exist",
          "app.containers.ViewQuestion.saveButton": "Save",
          "app.containers.ViewQuestion.theBest": "The best",
          "app.containers.ViewQuestion.theBestAnswer": "The best answer",
          "app.containers.ViewQuestion.title": "Question's page",
          "app.containers.ViewQuestion.voteToDelete": "Block",
          "app.containers.ViewQuestion.youHaveAnswers": "You cannot delete it because you already have some answers by your question",
          "app.containers.ViewQuestion.youVoted": "You have completed this action earlier",
          "app.containers.ViewQuestion.yourAnswer": "Your answer",
        }
      }
    />
  </div>
</React.Fragment>
`;

exports[`<ErrorBoundary /> snapshot test error is null 1`] = `
<React.Fragment>
  <div>
    Children
  </div>
</React.Fragment>
`;
